pkg/controller/controllercmd/builder.go

```diff
@@ -70,6 +70,9 @@ type ControllerBuilder struct {
 	authenticationConfig *operatorv1alpha1.DelegatedAuthentication
 	authorizationConfig  *operatorv1alpha1.DelegatedAuthorization
 	healthChecks         []healthz.HealthChecker
+
+	// nonZeroExitFn exists for unit testing
+	nonZeroExitFn func(args ...interface{})
```

Comment:
what do they do? comment

Reply:
`// ... called before terminating with non-zero exit code` ?

Reply:
@sttts this is a unit test stub (see comment)... so I'm able to test the Fatalf() behavior.

Reply:
what are args? If this is input for Sprintf, leave the format string as extra arg.

Reply:
@sttts this is not public API, right? only used in tests and private... I can surely improve the comments.

Reply:
@sttts improved godoc